DevOps.Util/HelixServer.cs

```diff
@@ -99,15 +108,45 @@ public async ValueTask GetHelixPayloads(string jobId, List<string> workItems, st
                     if (string.IsNullOrEmpty(workItemInfo.PayloadUri))
                         continue;
 
-                    await DownloadWorkitemFiles(workItemInfo.WorkItemId!, workItemInfo.PayloadUri).ConfigureAwait(false);
+                    string itemDir = await DownloadWorkitemFiles(workItemInfo.WorkItemId!, workItemInfo.PayloadUri).ConfigureAwait(false);
+
+                    Console.WriteLine();
+                    Console.WriteLine($"----- To execute: {workItemInfo.WorkItemId} -----");
+                    Console.WriteLine();
+                    string setPrefix = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? "set " : "";
+                    Console.WriteLine($"{setPrefix}HELIX_CORRELATION_PAYLOAD={correlationDir}");
+                    Console.WriteLine($"{setPrefix}HELIX_PYTHONPATH=echo skipping python");
```

Comment:
Bit of a hack here.  This makes the reporting portion of the helix scripts no-op.   I could find and rewrite them, but this was easier.

Reply:
Might consider also setting everything else you see in https://github.com/dotnet/arcade/blob/36d37f17dfca591203c5ddcad5a31f679e91d4a7/src/Microsoft.DotNet.Helix/Sdk/Readme.md#common-helix-client-environment-variables ... they're less commonly used but some are and it makes the tool more general purpose.